spec/ics-007-tendermint-client/README.md

```diff
@@ -128,10 +128,10 @@ Tendermint client validity checking uses the bisection algorithm described in th
 function checkValidityAndUpdateState(
   clientState: ClientState,
   header: Header) {
-    // assert trusting period has not yet passed
+    // assert trusting period has not yet passed. This should fatally terminate a connection.
```

Comment:
Right now, if the trusting period has passed, it will be impossible to update a client, so therefore impossible to use any connections associated with that client. Some sort of governance-based state recovery would be necessary.